assets/scss/checkout.scss

```diff
@@ -1,3 +1,85 @@
 // =============================================================================
 // Stencil Checkout  - Customize the checkout experience
 // =============================================================================
+
+// Main content
+.ng-checkout-container {
```

Comment:
Ah, right. I think it's meant to be an extension of `.appContainer`, it's just a badly formed class.